projects/1.20/assets/roost-re-hatched/roost/lang/zh_cn.json

```
@@ -0,0 +1,81 @@
+{
+  "advancements.almost_flying.descr": "将一只栖鸟的最大滑翔距离提升至上限",
+  "advancements.almost_flying.title": "仿若翱翔",
+  "advancements.commanding.descr": "制作命令哨，管理栖鸟的实用工具",
+  "advancements.commanding.title": "听令！",
+  "advancements.delicious_part_2_electric_boogalo.descr": "一种对栖鸟而言更美味的小食",
+  "advancements.delicious_part_2_electric_boogalo.title": "美味的（第二部分）： 霹雳舞动",
+  "advancements.delicious.descr": "制作一份幼虫沙拉，适用于任何鸟的美味小吃",
+  "advancements.delicious.title": "美味的？",
+  "advancements.farming_grubs.descr": "放置你的首个幼虫巢",
+  "advancements.farming_grubs.title": "养殖幼虫",
+  "advancements.fastest_roost_in_the_west.descr": "将一只栖鸟的最大速度提升至上限",
+  "advancements.fastest_roost_in_the_west.title": "西部快栖者",
+  "advancements.flippin_stones.descr": "掀开一块扁石",
+  "advancements.flippin_stones.title": "掀开扁石",
+  "advancements.gettin_grubby.descr": "得到一只幼虫，它们通常藏在扁石下面",
+  "advancements.gettin_grubby.title": "肮脏的收获",
+  "advancements.heavy_duty_poultry.descr": "将一只栖鸟的最大血量提升至上限",
+  "advancements.heavy_duty_poultry.title": "重禽",
```

```suggestion
  "advancements.heavy_duty_poultry.title": "耐劳良禽",
```
建议，heavy duty算是指耐用牢靠的意思